assembl/static2/js/app/utils/httpRequestHandler.js

```diff
@@ -1,24 +1,46 @@
+import urljoin from 'url-join';
+import Cookies from 'js-cookie';
```

Comment:
So we have a few helper functions `cookie` and `getCookieItem` from `js/app/utils/globalFunctions.js`. My thinking is, either use them, or we do a small refacto and use `js-cookie` everywhere. This package looks really small and heavily tested across all browsers, so I'm not opposed to it. The front-end code is hairy as is, and this just adds another layer of confusion for a front-end dev to decide what to use to access/manipulate cookies.

Reply:
I personally think the cookie stuff we're going in global functions is hairy and 'old school':

`export function setCookieItem(name: string, value: any, sensitive: boolean = false) {
  const date = new Date();
  date.setMonth(date.getMonth() + 13);
  document.cookie = cookie(name, value, date.toString(), sensitive);
}``

I don't see the need to create proprietary functions for things that have been simplified with packages, unless there's something I'm missing - I HATE it's accessing the document in our react code - I've never done this or felt the need to in my life - it's a hack.

Reply:
Not sure if accessing `document` from react is a hack at all. Even the js-cookie accesses it. I just read the source of `js-cookie`. Let's keep it, because it makes further checks on what's in the cookie, which we would have to implement in the future should we keep our code. However, might not be a bad idea to write a TODO comment in the `globalFunctions` to migrate these funcs to use `js-cookie`.